src/device-manager/java/src/nl/Weave/DataManagement/ResourceIdentifier.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ *    Copyright (c) 2019 Google LLC
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package nl.Weave.DataManagement;
+import android.os.Build;
+import android.util.Log;
+
+import java.math.BigInteger;
+/**
+ *  Represents Resource Identifier
+ */
+
```

Comment:
Move line break to between the import statement and Javadoc start?

Reply:
I might have done it in latest code, please help recheck, thanks

Reply:
Looks good, thanks